### PR TITLE
Release for CaaSP v4.5.4

### PR DESCRIPTION
**Pull request by davidcassany**
Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

File: adoc/MAIN.release-notes.adoc [hunk 1]
```diff
@@ -37,6 +37,7 @@ This release supports deployment on:
 
 
 // Release specific files, latest on top!
+include::release-4.5.3.adoc[Release 4.5.4]
```
Comment (kkaempf) [inline adoc/MAIN.release-notes.adoc]: Shouldn't that be `release-4.5.4.adoc` instead ?

File: adoc/release-4.5.4.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
+== Changes in 4.5.4
+
+=== New release of addon images
+
+4.5.4 brings a new release of addon images.
```
Comment (kkaempf) [inline adoc/release-4.5.4.adoc]: What has changed in the addon images ?

File: adoc/MAIN.release-notes.adoc (hunk 1 above)
Comment (davidcassany) [reply adoc/MAIN.release-notes.adoc]: :facepalm: yes you are right

File: adoc/release-4.5.4.adoc (hunk 2 above)
Comment (davidcassany) [reply adoc/release-4.5.4.adoc]: Right, I added that they include SLES patches up to the end of April 2021. Thanks

**Pull request merged**